valiu-node-rpc/tests/integration_tests/add_bob_as_a_member.rs

```diff
@@ -0,0 +1,21 @@
+use sp_core::sr25519::Pair;
+use sp_keyring::AccountKeyring;
+use substrate_subxt::{extrinsic::PairSigner, sudo::SudoCallExt, Client};
+use valiu_node_rpc::{AddMemberCall, ValiuRuntime};
+
+// This test will always pass because the `pallet-membership`' storage is not exposed, therefore,
```

Comment:
Perhaps we want to expose it? can we? There's even an item in the VLN API [Liquidity board](https://github.com/valibre-org/vln-api/projects/4) to query the list of providers

Reply:
We can either expose the membership storage that provides two methods (`members` and `prime`) or keep the storage hidden and create a new method on the `liquidity-provider` module to only expose the inserted members

Reply:
Exposing through `liquidity-provider` sounds like the way to go 🚀